packages/gestalt/src/Checkbox.js

```diff
@@ -29,29 +29,34 @@ type Props = {|
   size?: 'sm' | 'md',
 |};
 
-function Checkbox({
-  checked = false,
-  disabled = false,
-  errorMessage,
-  forwardedRef,
-  hasError = false,
-  id,
-  indeterminate = false,
-  label,
-  name,
-  onChange,
-  onClick,
-  size = 'md',
-}: Props): React.Node {
+function Checkbox(props: Props): React.Node {
+  const {
+    checked = false,
+    disabled = false,
+    errorMessage,
+    forwardedRef,
+    hasError = false,
+    id,
+    indeterminate = false,
+    label,
+    name,
+    onChange,
+    onClick,
+    size = 'md',
+  } = props;
   const inputElement = React.useRef<?HTMLInputElement>(null);
   const [focused, setFocused] = React.useState(false);
   const [hovered, setHover] = React.useState(false);
 
   React.useEffect(() => {
-    if (inputElement && inputElement.current) {
+    // $FlowFixMe
+    if (forwardedRef && forwardedRef.current) {
+      // $FlowFixMe
```

Comment:
how do I fix this Flow issue?
![image](https://user-images.githubusercontent.com/10593890/87739983-f3a5b800-c795-11ea-9502-a57b57ab0fa0.png)